vendor/brave-ios/Ledger/BATBraveLedger.h

```diff
@@ -188,8 +188,9 @@ NS_SWIFT_NAME(BraveLedger)
 
 @property (nonatomic, readonly) NSDictionary<NSString *, BATBalanceReportInfo *> *balanceReports;
 
-- (BATBalanceReportInfo *)balanceReportForMonth:(BATActivityMonth)month
-                                           year:(int)year;
+- (void)balanceReportForMonth:(BATActivityMonth)month
+                              year:(int)year
```

Comment:
Minor nit: Obj-C formatting, colon's should be aligned on new lines (Xcode would do this, Visual Studio Code probably doesnt know how)

Reply:
fixed